ppr-ui/src/views/newMhrRegistration/HomeOwners.vue

```diff
@@ -1,18 +1,205 @@
+              <hr class="my-8" />
+              <h3 class="text-center">Help with Owners</h3>
+              <h4>Sole Ownership</h4>
+              <p>
+                This applies when the home is owned by a single individual or
+                organization.
+              </p>
+              <h4>Joint Tenancy</h4>
+              <p>
+                This applies when the home is jointly owned by a number of
+                individuals or organizations or some combination of the two.
+              </p>
+              <h4>Tenants in Common</h4>
+              <p>
+                This applies when the home is owned by a number of groups or
+                individuals or organizations or some combination of the two
+                (where a group could consist of a single owner) and each group
+                of owners has the right to dispose of their share independent of
+                the other owner groups and will be disposed of as part of the
+                estate in the case of a death.
+              </p>
+              <hr class="my-8" />
+            </v-expansion-panel-content>
+          </v-expansion-panel>
+        </v-expansion-panels>
+      </div>
+
+      <label class="generic-label">
+        Your registration must include the following:
+      </label>
+      <div class="mt-5 mb-11">
+        <v-icon
+          v-if="getMhrRegistrationHomeOwners.length > 0"
+          color="green darken-2"
+        >
+          mdi-check
+        </v-icon>
+        <v-icon v-else color="black">mdi-circle-small</v-icon>
+        At least one owner
+      </div>
+      <v-btn
+        outlined
+        color="primary"
+        :ripple="false"
+        :disabled="
+          showAddPersonSection ||
+            showAddPersonOrganizationSection ||
+            isEditingMode
+        "
+        @click="showAddPersonSection = true"
+      >
+        <v-icon class="pr-1">mdi-account-plus</v-icon> Add a Person
+      </v-btn>
+
+      <span class="mx-2"></span>
+
+      <v-btn
+        outlined
+        color="primary"
+        :ripple="false"
+        :disabled="
+          showAddPersonOrganizationSection ||
+            showAddPersonSection ||
+            isEditingMode
+        "
+        @click="showAddPersonOrganizationSection = true"
+      >
+        <v-icon class="pr-1">mdi-domain-plus</v-icon>
+        Add a Business or Organization
+      </v-btn>
+
+      <div class="my-6">Tenancy Type: {{ tenancyType }}</div>
+    </section>
+
+    <v-expand-transition>
+      <AddHomeOwnerPerson
+        v-if="showAddPersonSection"
+        @done="addHomeOwner($event)"
+        @cancel="showAddPersonSection = false"
+      />
+    </v-expand-transition>
+
+    <v-expand-transition>
+      <AddHomeOwnerOrganization
+        v-if="showAddPersonOrganizationSection"
+        @done="addHomeOwner($event)"
+        @cancel="showAddPersonOrganizationSection = false"
+      />
+    </v-expand-transition>
```

Comment:
Same comments as before, believe we should consolidate our AddEdit Home Owner types.

Reply:
Refactored.